packages/table/src/table/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent } from 'react';
+import { PropertyFilter, Table as AWSUITable } from '@awsui/components-react';
+import { useCollection } from '@awsui/collection-hooks';
+import { TableProps } from '../utils';
+import { defaultI18nStrings, getDefaultColumnDefinitions } from '../utils/tableHelpers';
+
+export const Table: FunctionComponent<TableProps> = (props) => {
+  const { items, useCollectionOption = { sorting: {} }, columnDefinitions } = props;
+  const { collectionProps, propertyFilterProps } = useCollection(items, useCollectionOption);
+  return (
+    <AWSUITable
+      className="iot-table"
```

Comment:
does this class do anything? className will simply be ignored, `class` is the correct keyword. The className is a react concept, and peoples IDEs will sometimes automatically convert to `className` when you use JSX/TSX. But in general don't add classes that aren't necessary

Reply:
Agreed. I was trying reuse sc-table's styles but abandoned that idea later. Removing it.